velox/exec/OutputBuffer.h

```diff
@@ -272,6 +343,8 @@ class OutputBuffer {
   int32_t nextArbitraryLoadBufferIndex_{0};
   // One buffer per destination.
   std::vector<std::unique_ptr<DestinationBuffer>> buffers_;
+  // One bufferStat per destination.
```

Comment:
Redundant comment. It would be helpful to explain why we need this member variable and cannot just use buffers_[i].stats(). I assume this variable caches final stats for deleted buffers. Is this right? The number of entries in buffers_ and bufferStats_ is the same, but bufferStats_[i] is set only if buffers_[i] is null and vice versa.

Perhaps, finishedBufferStats_ would be a more descriptive name.

Reply:
done.